tests/toranj/test-010-on-mesh-prefix-config-gateway.py

```diff
@@ -207,7 +207,7 @@ def check_r1_is_associated():
 
 r1.add_prefix(
     prefix4,
-    48,
+    64,
```

Comment:
slaac is not enabled for this prefix, so I don't think we need to change its length to 64?

Reply:
I think we want to keep this one as 48 to verify behavior with non-slaac prefix with smaller prefix length.